R_scripts/weatherdat.R

```diff
@@ -5,13 +5,75 @@ library(readr)
 library(dplyr)
 library(ggplot2)
 library(ggrepel)
+library(lubridate)
+library(tidyr)
+
+# Function to replace S/N with descriptive label
+read_wxdat <- function(file) {
+  subfile <- read_csv("../inventory_data/wstation_info.csv")
+  wdat <- file %>% 
+    read_csv(skip=1) %>%
+    gather(label, Value, -1:-2) %>% 
```

Comment:
A couple comments would be useful for the above lines, e.g. "Drop first two columns, which contain X and Y"